lib/streamlit/runtime/secrets.py

```diff
@@ -150,6 +216,19 @@ def load_if_toml_exists(self) -> bool:
             # No secrets.toml files exist. That's fine.
             return False
 
+    def set_print_exceptions_on_access(self, print_exceptions_on_access: bool) -> None:
```

Comment:
When is this called?

Reply:
I'll explain in DM - in some environments we want to not fire exceptions if no secret files are found.